### DISCUSSION TITLE
How do you export goxel files with colours into unity?

### DISCUSSION THREAD

**Discussion by JasperMoerland** [guillaumechereau/goxel#338] [Q&A] [answered]
I have already tried to use the materials it did not work.

**Comment by guillaumechereau** [comment]
Can you attach a 'gox' file that does not work?
What format do you export in?  The recommended format is glTF with texture (no vertex color).

Also, can you try with the last build from GitHub CI, since I did some changes recently that have not be released yet:
https://github.com/guillaumechereau/goxel/actions/runs/7430263205

**Reply by JasperMoerland** [reply]
I export in obj, and I am a macos user, with an m1 chip. What is a texture and a vertex color?

**Comment by guillaumechereau** [comment, marked as answer]
I see.  Currently the obj export format does not support texture so the
colors are set at each vertices, which does not work well with unity i
guess.  Can you instead export in gltf ?

**Reply by JasperMoerland** [reply]
Yes thank you, I can now continue to use your software (It is 1000X better than magicavoxel)

**Reply by JasperMoerland** [reply]
also I used blender to convert it from gltf to obj
